versiondb/client/ingest_sst.go

```diff
@@ -21,7 +21,12 @@ func IngestVersionDBSSTCmd() *cobra.Command {
 				return err
 			}
 
-			db, cfHandle, err := tsrocksdb.OpenVersionDB(dbPath)
+			opts := tsrocksdb.NewVersionDBOpts(false)
+			// it's a workaround because rocksdb always ingest files into level `num_levels-1`,
+			// the new data will take a very long time to reach that level,
```

Comment:
Is the conclusion base on the mainnet data sets?

Reply:
level 2: 5G
level 3: 50G

testnet is around lower bound of level 3, mainnet is around upper bound of level 3